nengo/tests/test_connection.py

```diff
@@ -611,6 +612,21 @@ def test_set_eval_points(Simulator):
     Simulator(model)  # Builds fine
 
 
+@pytest.mark.parametrize('radius', [0.5, 1., 1.5])
+def test_eval_points_scaling(Simulator, radius, seed, rng):
+    eval_points = UniformHypersphere().sample(500, 1, rng=rng)
```

Comment:
Can Connection not take a Distribution for `eval_points`. Just glancing at the builder code, it looked like maybe it can't. We should probably fix that. And if and when it can, we should probably have this test give both a distribution and a bunch of points, just like the test for Ensemble.